aten/src/ATen/cpu/vec256/vec256_int.h

```diff
@@ -613,6 +877,24 @@ Vec256<int16_t> inline operator*(const Vec256<int16_t>& a, const Vec256<int16_t>
   return _mm256_mullo_epi16(a, b);
 }
 
+template <typename T, typename Op>
+Vec256<T> inline int_elementwise_binary_256(const Vec256<T>& a, const Vec256<T>& b, Op op) {
+  T values_a[Vec256<T>::size()];
```

Comment:
Should these 2 lines be \_\_at_align32\_\_?

Reply:
Yes, I've added them here. I think the bigger danger is that `storeu` as defined for integer Vec256 classes requires the parameter to be aligned (e.g., see https://github.com/pytorch/pytorch/blob/3e6bb5233f9ca2c5aa55d9cda22a7ee85439aa6e/aten/src/ATen/cpu/vec256/vec256_int.h#L452), but in the code there is no static check. Do you agree? If so, I will be open an issue and find an opportunity to fix this unprotected behavior.

Reply:
@xuhdev 
Yep, this is exactly the place I've looked at. Let's open an issue then. Please cc @VitalyFedyunin @ezyang .